modules/siddhi-extensions/timeseries/src/test/java/org/wso2/siddhi/extension/timeseries/KalmanExtensionTestCase.java

```diff
@@ -485,4 +489,78 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
 
     }
 
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void testKalmanMinStreamProcessorExtension1() throws InterruptedException {
+        log.info("KalmanMin TestCase with test attributeExpressionExecutor's length");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String inStreamDefinition = "define stream inputStream (id int ,price double);";
+        String query = ("@info(name = 'query1') from inputStream#timeseries:kalmanMinMax(price, 0.000001,0.0001, "
+                + "'min') select price, extremaType, id insert into outputStream;");
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(inStreamDefinition +
+                                                                                                     query);
+        executionPlanRuntime.start();
```

Comment:
No need to start and stop in parameter validation testcases. Please remove all instances.